pandas/formats/format.py

```diff
@@ -903,8 +917,50 @@ def get_col_type(dtype):
                          if x else '{}') for x in row]
             else:
                 crow = [x if x else '{}' for x in row]
+            if i < nlevels and self.fmt.header and self.multicol:
+                row2 = list(crow[:ilevels])
+                ncol = 1
+                coltext = ''
+
+                def appendCol():
+                    if ncol > 1:
+                        row2.append('\\multicolumn{{{0:d}}}{{c}}{{{1:s}}}'
+                                    .format(ncol, coltext.strip()))
+                    else:
+                        row2.append(coltext)
+                for c in crow[ilevels:]:
```

Comment:
Without the multicolumn feature, long names in higher levels break the layout severely:
![nocolmultirow](https://cloud.githubusercontent.com/assets/8004308/18923834/23a367ba-85ae-11e6-8ba9-f0c254e36f08.png)
compared to:
![multicolmultirow](https://cloud.githubusercontent.com/assets/8004308/18923833/238f679c-85ae-11e6-946a-50ab84c10e4b.png)
Thus, i would propose this feature to be enabled by default.
This means, that empty columns won't be printed, which may alter the code of the line with the row-names.
Not the result though, as latex ignores these trailing empty parts anyways.

Should i rather set the default to False and maintain compatibility in code, or fix the rendered output by rewriting fixing tests?
Or has somebody an idea to satisfy both?